packages/renderer/MdToHtml.ts

```diff
@@ -399,9 +399,10 @@ export default class MdToHtml implements MarkupRenderer {
 	public async allAssets(theme: any, noteStyleOptions: NoteStyleOptions = null) {
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
 		const assets: any = {};
-		for (const key in rules) {
+		const allRules = { ...rules, ...this.extraRendererRules_ };
+		for (const key in allRules) {
```

Comment:
This change (along with a few other modifications) add support for loading plugin-specified assets in the Rich Text Editor. This allows migrating inline event handlers to JavaScript files that aren't blocked by the content security policy. Plugin-specified assets were previously only supported in the Markdown viewer.